src/main/resources/templates/fragments/icons.html

```diff
@@ -1967,6 +1967,84 @@ <h1>Icons</h1>
         </svg>
       </td>
     </tr>
+    <tr>
+      <td>submitAdditionalIcons</td>
```

Comment:
We haven't put the word "Icons" in the names so far.   What is this icon of?  The name doesn't make it clear -- should be what it is, not what it's for.

Reply:
Also, this is a file, so it's probably better to name it "file" or something relating to what it looks like, not what we are using it for in the application.  That way others can get an idea of what it looks like and use it as they please.

Reply:
Actually, I take this back about the word "Icon". We have a varied practice now with the word "Icon", so what you did is not inconsistent with what's in the file, but I think we should take the word Icon out from all spots.

Reply:
Updated to: smallFile